realm/realm-library/build.gradle

```diff
@@ -65,6 +65,16 @@ android {
                 }
             }
         }
+
+        buildTypes {
+            debug {
+                javaCompileOptions {
+                    annotationProcessorOptions {
+                        arguments = [ debug : 'true' ]
```

Comment:
Would `suppressWarnings` with a value of `false` be more readable? I feel a bit uneasy about using the word `debug` since it implies a lot more. At least in my mind?